### DISCUSSION TITLE
Implement Dark Mode with One Line of CSS | /en/2023/09/dark-mode/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1519] [Comments]
Personally I do not like the dark mode. I love sharp full-black text on clean white background, which gives me the same feeling as reading on paper. Reading in the dark mode makes me feel lost in the &hellip;

https://yihui.org/en/2023/09/dark-mode/

**Comment by binkybarnes** [comment]
wow cool

**Comment by crowjake** [comment]
Nice! As you mention the colors will be inverted you can avoid this partially, by changing `filter: invert(1)` to `filter: invert(1) hue-rotate(180deg);` and the correct hue will be restored (retaining inverted brightness)

**Reply by yihui** [reply]
Thank you for the tip!
